key.go

```diff
@@ -382,10 +382,10 @@ var sequences = map[string]Key{
 	"\x1b[2;3~":   {Type: KeyInsert, Alt: true},
 	"\x1b[3~":     {Type: KeyDelete},
 	"\x1b[3;3~":   {Type: KeyDelete, Alt: true},
-	"\x1b[1~":     {Type: KeyHome},
```

Comment:
While you're right and most every terminal seems to send `\x1b[H` for the home key, I do find references to this sequence being as home as well. We should probably keep it?

Reply:
Same question/remark for `KeyEnd`, too.

Reply:
On which terminal was that?

Reply:
I'll update this commit and keep the previous sequences for `KeyHome` and `KeyEnd` for now. If we can confirm they aren't used anywhere we can still remove them eventually.

Reply:
Ok, pushed my changes to your fork.

Reply:
Ok this works for me. thanks for the edit!